docs/team/benjytan45678.md

```diff
@@ -2,34 +2,47 @@
+    * Justification: This feature allows a user to pin certain applicants with special circumstances (such as possessing any form of disability or currently still undergoing National Service), 
+    so that the user will be able to refer to these set of applicants at a later stage of time after receiving some form of clearance from relevant authorities.
 
-* **Code contributed**: 
-* Refer to https://github.com/AY2223S1-CS2103T-W10-3/tp/tree/master
-* and https://github.com/AY2223S1-CS2103T-W10-3/tp/pulls?page=1&q=is%3Apr+author%3Abenjytan45678
+* **Code contributed**: Refer to my [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=benjy&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=benjytan45678&tabRepo=AY2223S1-CS2103T-W10-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
 * **Project management**:
+  * Setting up the GitHub team organisation/repository [#25](https://github.com/nus-cs2103-AY2223S1/tp/pull/25)
+  * Setting up tools e.g., GitHub, Gradle and CodeCov
+  * Helped to handle the creation of milestone v1.2 and creation of team issues.
 
 * **Enhancements to existing features**: 
-* Added an `AlertWindow` class under `ui` for both `clear` and `remove` commands
-* Implemented `PinCard`, `PinList` and its css files improve the user interface of the GUI
-* Added `StatusPredicate` class under `model` with Jie Hui.
+  * Enhanced the `remove` and `clear` commands with an alert dialog box. [#83](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/83/files) and [#90](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/90)
+    * What it does: Receives user's confirmation before proceeding with the removal or clearing of data from TrackAScholar.json
+    * Justification: Since the above commands are irreversible in nature, the alert dialog box serves as a deterrence against unintentional usage of the commands, thus improving the overall experience of the product for the user
+    * Highlights: Still optimized for fast typists as the prompt can be answered using keyboard commands
+  * Enhanced the GUI of TrackAScholar. [#101](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/101/files) and [#185](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/185)
+    * Implemented `PinCard`, `PinList` and its css files to improve the overall user interface.
 
 * **Documentation**: 
   * User Guide: Added unpin command in features.
   * Developer Guide: Added the filter command implementation, filter command
-    sequence diagram, filter command activity diagram and Use Case 3.
+    sequence diagram, filter command activity diagram and Use Case 3. [#96](https://github.com/AY2223S1-CS2103T-W10-3/tp/pull/96/files)
```

Comment:
Maybe u can add the details for documentation on User Guide and Developer Guide in a point form under the User Guide. i.e.

* **Documentation**
    * User Guide:
        * Added documentation for `unpin` [#...]()

etc and remember to link them to ur respective PR that u wrote it on